src/main/java/org/opensearch/ad/stats/suppliers/ModelsOnNodeSupplier.java

```diff
@@ -43,13 +44,17 @@
 import org.opensearch.ad.caching.CacheProvider;
 import org.opensearch.ad.constant.CommonName;
 import org.opensearch.ad.ml.ModelManager;
+import org.opensearch.cluster.service.ClusterService;
+import org.opensearch.common.settings.Settings;
 
 /**
  * ModelsOnNodeSupplier provides a List of ModelStates info for the models the nodes contains
  */
 public class ModelsOnNodeSupplier implements Supplier<List<Map<String, Object>>> {
     private ModelManager modelManager;
     private CacheProvider cache;
+    // the max number of models to return per node. Defaults to 10.
+    private volatile int modelToReturn;
```

Comment:
Suggest to change to `modelsToReturn` for clarity that this is plural

Reply:
Or even `numModelsToReturn`. Let me know what you think

Reply:
changed to use numModelsToReturn